packages/components/tests/acceptance/components/hds/alert-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | components/hds/alert', function (hooks) {
```

Comment:
```suggestion
module('Acceptance | components | /hds/alert', function (hooks) {
```
for consistency with the accordion test